sandbox/index.html

```diff
@@ -58,6 +58,20 @@
       right: 1em;
     }
   </style>
+  <script type="importmap">
+    {
+      "imports": {
+        "lit": "../node_modules/lit/index.js",
+        "lit/": "../node_modules/lit/",
+        "@lit/reactive-element": "../node_modules/@lit/reactive-element/reactive-element.js",
+        "@lit/reactive-element/": "../node_modules/@lit/reactive-element/",
+        "lit-html": "../node_modules/lit-html/lit-html.js",
+        "lit-html/": "../node_modules/lit-html/",
+        "lit-element": "../node_modules/lit-element",
+        "lit-element/lit-element.js": "../node_modules/lit-element/lit-element.js"
+      }
+    }
```

Comment:
**Question**: As discussed in sync, do we keep this or do we move the sandboxes somewhere inside `src/stories`?

We should probably also open an RFC to discuss how we want to be able to test our components from now on but I still need a quick solution / decision so we don't block this PR :thinking: